README.md

```diff
@@ -49,6 +51,31 @@ generator.write_to "jsondoc"
 Without this configuration, the `source` URLs for the code on GitHub will be
 incorrect. You don't need it if `lib` is in the repo root.
 
+#### Generating a TOC resource (page)
+
+You can generate a resource (page) for the Angular site app with a special
+description containing a TOC table for select classes/modules. The `package`
+property will be used for the headline of the description. The `include`
+property is a regex matched to jsondoc filepaths:
+
+```ruby
+require "gcloud/jsondoc"
+
+registry = YARD::Registry.load! ".yardoc"
+
+toc_config = {
+  types: [
+    {
+      title: "Google::Datastore::V1::DataTypes",
+      toc: { package: "Google::Datastore::V1", include: "google/datastore/v1/" }
```

Comment:
This works well for Datastore, because `Google::Cloud::Datastore::V1` and `Google::Datastore::V1` are separate namespaces, but for other services the GAPIC code and Protobuf code share the same namespace. How will Jsondoc differentiate between the two?